core/com.b2international.snowowl.core/src/com/b2international/snowowl/core/date/EffectiveTimes.java

```diff
@@ -157,7 +156,12 @@ public static final long getEffectiveTime(@Nullable Date effectiveTime) {
 		if (null == effectiveTime) {
 			return UNSET_EFFECTIVE_TIME;
 		}
-		return effectiveTime.getTime();
+		return effectiveTime.toInstant()
+			      .atZone(ZoneId.of("GMT"))
```

Comment:
For consistency:

```suggestion
			      .atZone(ZoneOffset.UTC)
```

Reply:
Actually, this is how it looked like initially, but I intentionally changed it, since one requires a ZoneId and the other requires a ZoneOffset, anyway, the logic will remain the same, so happy to change it back. 😄